Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
But the caveats can't robustly be made dependent on whether this version of the formula was installed fresh, or as an upgrade from a version old enough that it made the transition.

I'd say switch this back to `etc.install` and don't bother with caveats; users will be able to figure it out.